src/Tizen.Applications.ComponentBased.Default/Tizen.Applications.ComponentBased.Default/ElmWindow.cs

```diff
@@ -0,0 +1,55 @@
+﻿using ElmSharp;
+using System;
+using Tizen.Applications.ComponentBased.Common;
+
+namespace Tizen.Applications.ComponentBased.Default
+{
+    /// <summary>
+    /// Window class for basic application type
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ElmWindow : IWindow
```

Comment:
X DO NOT use abbreviations or contractions as part of identifier names.

```suggestion
    public class ElementaryWindow: IWindow
```
```suggestion
    public class EFLWindow: IWindow
```

Reply:
Done